xbmc/cores/dvdplayer/DVDPlayer.cpp

```diff
@@ -3318,6 +3318,24 @@ bool CDVDPlayer::OpenVideoStream(CDVDStreamInfo& hint, bool reset)
     }
     hint.software = true;
   }
+  else if (m_pInputStream && m_pInputStream->IsStreamType(DVDSTREAM_TYPE_PVRMANAGER))
+  {
+    // set framerate if not set by demuxer
+    if (hint.fpsrate == 0 || hint.fpsscale == 0)
+    {
+      int fpsidx = CSettings::Get().GetInt("pvrplayback.fps");
+      if (fpsidx == 0)
+      {
+        hint.fpsscale = 1000;
+        hint.fpsrate = 50000;
+      }
+      else if (fpsidx == 1)
+      {
+        hint.fpsscale = 1000;
+        hint.fpsrate = 59940;
+      }
```

Comment:
Because 59.94 <> 60000/1001? :) And that's the true framerate of NTSC. I see the comment but it's also wrong IMO. The whole purpose of expressing framerates with two numbers is to be able to precisely refer to these awkward rates (60000/1001, 24000/1001). Otherwise a float could suffice.

Not sure where these hints will be used. If for display only then it does not matter much. If for timing then higher accuracy is better.